test-e2e/fixtures/apps/render-reasons-memo.jsx

```diff
@@ -0,0 +1,32 @@
+import { h, render } from "preact";
+import { memo } from "preact/compat";
+import { useState } from "preact/hooks";
+
+function Inner() {
+	return <p>Inner</p>;
+}
+
+function Foo() {
+	return <Inner />;
+}
+
+const MemoFoo = memo(Foo);
+
+function AppInner(props) {
+	return props.children;
+}
+
+function App() {
+	const [v, set] = useState(0);
+
+	return (
+		<AppInner>
+			<div>
+				<button onClick={() => set(v + 1)}>Update</button>
+				<MemoFoo />
```

Comment:
Sorry, I found an issue when we include another memorized component that iterates over an array to return a more complex output (list of elements). 

https://github.com/preactjs/preact-devtools/pull/359